scripts/install_zig.sh

```diff
@@ -68,6 +69,7 @@ if command -v wget; then
     if [ -f /etc/alpine-release ]; then
 	ipv4=""
     fi
+    # shellcheck disable=SC2086 # We control ipv4 and it'll always either be empty or -4
     wget $ipv4 --quiet --output-document="$ZIG_TARBALL" "$ZIG_URL"
```

Comment:
Why are these shellcheck notes only now needed?

Reply:
Good question - it was throwing errors for me, and now I see removing them it's fine, perhaps I had another change that made it not able to infer `ipv4` was fixed.